zenduty/README.md

```diff
@@ -0,0 +1,65 @@
+## Overview
+
+Use the Zenduty integration to send Datadog alerts to the right team, notify them as per on-call schedules, and help them remediate and resolve incidents with speed.
```

Comment:
Can we clarify where the alerts are being sent? Are the Datadog alerts being sent within Zenduty or within Datadog?

Reply:
Done!